bootstrap/httpserver/server.go

```diff
@@ -34,7 +34,7 @@ func NewHttpServer(address string, reporting instrumentation.Reporting, publicAp
 	}
 
 	go func() {
-		reporting.Infof("starting server on address %s", address)
+		reporting.Info("Starting server on address", String("address", address))
```

Comment:
I need to understand what we're trying to achieve with have this typed logging.

Reply:
I think that the idea is structured logging. Not sure that this _specific_ log entry is a representative example, though :)